Universal/brg0.md

```diff
@@ -0,0 +1,17 @@
+# Fixing Property Injection (SSDT-BRG0)
+
+* [Fixing Property Injection (SSDT-BRG0)](#fixing-property-injection-ssdt-brg0)
+  * [What this SSDT does](#what-this-ssdt-does)
+  * [Methods to make this SSDT](#methods-to-make-this-ssdt)
+
+## What this SSDT does
+
+On some platforms, the PCI bridge your GPU is connected to is not defined.
+This causes DeviceProperties from your config to fail to inject.
```

Comment:
```suggestion
This can prevent Device Properties being added to these devices from your config.plist.
```
I think this might work a bit better, but feel free to rework.